src/julia.h

```diff
@@ -1839,8 +1840,8 @@ JL_DLLEXPORT jl_sym_t *jl_symbol_n(const char *str, size_t len) JL_NOTSAFEPOINT;
 JL_DLLEXPORT jl_sym_t *jl_gensym(void);
 JL_DLLEXPORT jl_sym_t *jl_tagged_gensym(const char *str, size_t len);
 JL_DLLEXPORT jl_sym_t *jl_get_root_symbol(void);
-JL_DLLEXPORT jl_value_t *jl_get_binding_value(jl_binding_t *b JL_PROPAGATES_ROOT) JL_NOTSAFEPOINT;
-JL_DLLEXPORT jl_value_t *jl_get_binding_value_if_const(jl_binding_t *b JL_PROPAGATES_ROOT) JL_NOTSAFEPOINT;
+JL_DLLEXPORT jl_value_t *jl_get_binding_value(jl_binding_t *b JL_PROPAGATES_ROOT);
+JL_DLLEXPORT jl_value_t *jl_get_binding_value_if_const(jl_binding_t *b JL_PROPAGATES_ROOT);
```

Comment:
Both of these changes violate JL_NOTSAFEPOINT requirements in rtutils and codegen (the two files we aren't able to verify). I think you partly fixed one of the issues, but not the others

Reply:
I fixed rtutils. I don't think codegen assumes JL_NOTSAFEPOINT, but regardless these will be removed from codegen anyway as it gets adjusted to partition.

Reply:
Yes, I saw the one place was fixed, but looks like `jl_stderr_obj` still needs to be deleted still from rtutils, as it is now invalid (and hasn't been used in many years anyways). The annotations on jl_declare_constant_val also still need to be fixed, as it looks like we now violate them, and pass in unrooted objects (var) then call the allocation function